mifospay/src/main/java/org/mifos/mobilewallet/mifospay/auth/ui/LoginActivity.java

```diff
@@ -214,4 +222,12 @@ public void onComplete(@NonNull Task<Void> task) {
                     });
         }
     }
+
+    private void hideSoftKeyboard() {
+        View view = this.getCurrentFocus();
+        if(view != null) {
+            InputMethodManager inputMethodManager = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
```

Comment:
Your build is failing because of code in line number 229. This line of code contains more than 100 chars, which is a PMD violation. To make your build pass break the line into half.

Reply:
Hey, thanks. I overlooked this. :)

Reply:
Looks like that wasn't it. Anyway, I'll look for it.